Bluepill-runner/Bluepill-runner/BPApp.m

```diff
@@ -98,13 +105,23 @@ + (instancetype)appWithConfig:(BPConfiguration *)config
 
     BPApp *app = [[BPApp alloc] init];
     NSMutableArray<BPXCTestFile *> *allTests = [[NSMutableArray alloc] init];
-
+    NSDictionary *classMappings = nil;
+    if (config.inheritedClassMappingJsonFile) {
```

Comment:
Is this a maintainable solution? Shouldn't BP generate the class mapping file for accuracy?